onnxruntime/core/platform/posix/env.cc

```diff
@@ -230,7 +230,7 @@ class PosixThread : public EnvThread {
         } else {
           errno = ret;
           auto [err_no, err_msg] = GetErrnoInfo();
-#if !defined(USE_MIGRAPHX)
+#if !defined(USE_MIGRAPHX) && !defined(USE_MIGRAPHX_INTERFACE)
```

Comment:
I wonder if we can just remove these macros entirely since it's only wrapping a logging statement.